README.md

```diff
@@ -2,13 +2,10 @@
 - The Fenders Perth Meetup website.
 
 ## TODO
-- [x] Setup components
 - [ ] Convert CSS to Emotion or Styled Components
-- [ ] Pull Meetup events in from Meetup.com
 
 ## Running in development
-`gatsby develop`
-
+`MEETUP_API={API KEY} gatsby develop`
```

Comment:
Important

Reply:
Im pretty sure we can add the 'MEETUP_API' as an environment var in Netlify. Will have to get @amykapernick onto that one, or I'll get the login details.